atecc608a/atecc608a_utils.c

```diff
@@ -69,10 +70,11 @@ psa_status_t atecc608a_write_lock_config(const uint8_t *config_template,
     /* Copy 16 bytes of device-specific data to the prepared config buffer */
     ASSERT_SUCCESS(atcab_read_bytes_zone(ATCA_ZONE_CONFIG, 0, 0, config, 16));
 
-    atCRC(length, config, &crc);
+    atCRC(length, config, crc);
+    crc16 = (crc[1] << 8u) | crc[0];
```

Comment:
In "Fix crc type mismatch", add a C code comment explaining that `atCRC()` guarantees this endianness. Otherwise, it might appear to a casual reader that we are making assumptions about the machine endianness.

Reply:
Done